app/views/pregnancies/edit.html.erb

```diff
@@ -16,7 +16,7 @@
       <%= render partial: 'pregnancies/patient_information', locals: { f: f, pregnancy: @pregnancy } %>
       <%= render partial: 'pregnancies/change_log', locals: { f: f, pregnancy: @pregnancy } %>
       <%= render partial: 'pregnancies/call_log', locals: { pregnancy: @pregnancy } %>
-      <% end %>
+<% end %>
```

Comment:
hah. I had to look a bit to figure out what it went with and why there wasn't an error with a seemingly random floating `<% end %>`. But it seems reasonable with all the partials, though if there's a cleaner way to do it I'd be interested to see.